src/index.html

```diff
@@ -1,13 +1,96 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
+
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <title>Stars</title>
-    <link rel="stylesheet" href="./style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Stars Blocks</title>
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="stars stars--0">
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+      <div class="stars__star"></div>
+
+    </div>
+
+    <div class="stars stars--1">
```

Comment:
There should be empty lines between multiline sibling blocks of HTML but don't add empty lines between parent and child elements
```suggestion
    <div class="stars stars--0">
      <div class="stars__star"></div>

      <div class="stars__star"></div>

      <div class="stars__star"></div>

      <div class="stars__star"></div>

      <div class="stars__star"></div>
    </div>

    <div class="stars stars--1">
```